setupbase.py

```diff
@@ -153,7 +153,6 @@ def find_package_data():
         pjoin(components, "underscore", "underscore-min.js"),
         pjoin(components, "moment", "moment.js"),
         pjoin(components, "moment", "min", "moment.min.js"),
-        pjoin(components, "term.js", "src", "term.js"),
```

Comment:
Does it make sense to replace it with a check for xterm.js? Or does the bundling mean there isn't a separate file for it?